jdaviz/core/template_mixin.py

```diff
@@ -2953,8 +2953,7 @@ def _get_continuum(self, dataset, spectral_subset, update_marks=False, per_pixel
         if per_pixel:
             if self.app.config != 'cubeviz':
                 raise ValueError("per-pixel only supported for cubeviz")
-            full_spectrum = self.app._jdaviz_helper.get_data(self.dataset.selected,
-                                                             use_display_units=True)
+            full_spectrum = self.app._jdaviz_helper.get_data(self.dataset.selected)
```

Comment:
is this (and the other removal below) because the marks are handling the unit conversion themselves?

Reply:
also see https://jira.stsci.edu/browse/JDAT-4830, this bug was not introduced with this change